README.md

```diff
@@ -1,3 +1,5 @@
+![OVH component](githubBanner.png)
```

Comment:
I'm pretty sure that we can and should put this on a CDN instead of copy it on each repositories. @mlarose Do you think we should require to put this on a CDN before merging ?